### ISSUE TITLE
gcs_upload error: Cannot insert legacy ACL for an object when uniform bucket-level access is enabled

### ISSUE THREAD

**Issue by ndrubins** [cloudyr/googleCloudStorageR#121]
Hi,

I apologize if this is a usage or google cloud storage settings issue but I couldn't really find any relevant online documentation.

I'm simply trying to upload a file with this command:

`googleCloudStorageR::gcs_upload("test/my_test.csv")
`

And after these two messages:
```
2020-05-11 18:57:19 -- File size detected as 368 bytes
2020-05-11 18:57:20> Request Status Code: 400
```

I get this error:
`Error: API returned: Cannot insert legacy ACL for an object when uniform bucket-level access is enabled. Read more at https://cloud.google.com/storage/docs/uniform-bucket-level-access.
`

Any idea how to solve this?

**Comment by MarkEdmondson1234** [comment]
This happens if the bucket has all permissions set at bucket level and is solved in issue #111. It’s not on CRAN yet but you can install the GitHub version to fix it.

**Comment by ndrubins** [comment]
Thanks a lot
